online-shop/src/app/modules/shopping-cart/components/presentational/shopping-cart-details-view/shopping-cart-details-view.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, Input } from '@angular/core';
+import { Router } from '@angular/router';
+import { Product } from 'src/app/types/products.types';
+import { ProductAndProductCategoryDto } from 'src/app/types/productsAndProductsCategoryDto';
+
+@Component({
+  selector: 'app-shopping-cart-details-view',
+  templateUrl: './shopping-cart-details-view.component.html',
+  styleUrls: ['./shopping-cart-details-view.component.scss']
+})
+export class ShoppingCartDetailsViewComponent {
+  @Input()  productList :ProductAndProductCategoryDto[] | undefined;
+
+  constructor(private router: Router) {}
```

Comment:
I'd rather you use `routerLink="/"` in the HTML file rather than injecting the router and creating a method for this.